website/content/en/docs/ansible/development-tips.md

```diff
@@ -492,17 +520,17 @@ The structure passed to Ansible as extra vars is:
    },
 }
 ```
-`message` and `newParameter` are set in the top level as extra variables, and
-`meta` provides the relevant metadata for the Custom Resource as defined in the
-operator. The `meta` fields can be accesses via dot notation in Ansible as so:
+
+The `message` and `newParameter` at the top level are set as extra variables
+themselves. The `meta` key contains some metadata relevant to the Custom
+Resource. You can access these metadata  via dot notation in Ansible:
```

Comment:
```suggestion
Resource. The `meta` fields can be accesses via dot notation in Ansible as so:
```

Reply:
I came from a kubernetes docs community where there is a guide line [use active voice](https://kubernetes.io/docs/contribute/style/style-guide/#use-active-voice).
I'm trying to share some common practices from tech writers, though I am not strongly opinionated on this.

Reply:
Very interesting. I did not know that it has a definition. I agree we should follow up that. Make senses for me. 
Tks for share I will flag it internally as well.